eidas_proxy_service/tests/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""Tests of eidas_proxy_service."""
```

Comment:
I think that this can go away now, as we have tested travis and codecov.

Reply:
Dummy test case removed.